www/docs/intro.md

```diff
@@ -375,6 +375,14 @@ Devnet offers [empty block creation](https://shard-labs.github.io/starknet-devne
 const emptyBlock = await starknet.devnet.createBlock();
 ```
 
+#### Mint tokens to an account
+
+DevNet allows [minting token](https://shard-labs.github.io/starknet-devnet/docs/guide/mint-token#mint-with-a-transaction). You can call
```

Comment:
We could use "Devnet" instead of "DevNet" here.

Reply:
@shramee What about this?

Reply:
Aah, I thought we wanted some continuation assuming `You can call` followed by code example looked rather abrupt.
Fixed now.